### ISSUE TITLE
Vim motions or keybindins

### ISSUE THREAD

**Issue by Notnobcoder** [excalidraw/excalidraw#6925]
Can we add vim motions and key bindings for text edition

**Comment by ad1992** [comment]
@Notnobcoder please elaborate what do you mean ?

**Comment by dwelle** [comment]
We _may_ add vim motions later when switch to a richtext editor. But even then it seems a bit out of scope in the context of Excalidraw where most text are short diagram labels.

**Comment by Jhonathan-Pedroso** [comment]
Vim bindings would be absolutely neat

**Comment by vigasdeep** [comment]
@Jhonathan-Pedroso @Notnobcoder  would you guys like to provide an example of how Vim bindings can help here?

**Comment by niceiq** [comment]
Let me cook:

**Main use case:** For diagramming (would mostly use it when navigating and editing flowcharts)

i don't want to constantly have to reach for my mouse to create new nodes in the fllowchart. The flowchart update was well received and we now have bindings (CMD/CTRL + <arrow>) to create new nodes, but it's just a pain to use that short cut.

A nice-to-have would be:
- The ability to use (CMD/CTRL +h/j/k/l) to create new nodes (this is the thing I'd really like to have)
- A model interface to navigate nodes. For example, you can move around nodes in normal mode pressing h/j/k/l and hit `i` to  edit the current note you're at.
- (optional) vim bindings to edit the text itself

@dwelle @ad1992

**Comment by radouane-tamouss** [comment]
Adding Vim motion support for text editing in Excalidraw would be a fantastic feature! It seems others have already requested this as well, and I believe it would greatly enhance the workflow for many users.

**Comment by Jhonathan-Pedroso** [comment]
Yeah I think the ability to move between nodes, and start insert mode
within the vim binding; attached some background with line numbering as a
grid; would be nice to jump around and quickly create content;

**Comment by Vilhelm-Ian** [comment]
i just want to be able to change the camera position with j h k l
